### ISSUE TITLE
Image Artefacts after SDC correction

### ISSUE THREAD

**Issue by xuyangwen** [nipreps/fmriprep#2796]
### What happened?

We implemented SDC correction using fmriprep 21.02 using phase-difference B0 estimation and found very big artifacts. However, when we applied _fsl_prepare_fieldmap_ command in FSL on the same images using the same fieldmap (we collected SIEMENS data), there were no such big artifacts. Please see the attached screenshots.

### What command did you use?

```shell
/opt/conda/bin/fmriprep /data /out participant --participant-label sub-*** --write-graph --use-aroma -w /scratch --output-spaces MNI152NLin2009cAsym:res-native T1w:res-native
```

### What version of fMRIPrep are you running?

21.02

### How are you running fMRIPrep?

Docker

### Is your data BIDS valid?

Yes

### Are you reusing any previously computed results?

FreeSurfer

### Please copy and paste any relevant log output.

```shell
We do not have any errors reported.
```

### Additional information / screenshots

[Presentation1.pdf](https://github.com/nipreps/fmriprep/files/8891578/Presentation1.pdf)

**Comment by effigies** [comment]
Thanks for this. We're collecting extreme cases for SDC correction, and may ask you to share your data, if you're willing. Can you at least share a full file listing and the metadata of both your fieldmaps and your BOLD series?

**Comment by effigies** [comment]
Should be resolved in 23.2.0a1.
